fixing the tutorials

lavocatt opened the pull request:
This fixes the tutorials with two commits:

```
    [#1264] make use of forceSSLParameters
    
    On some environments JAVA env variables can conflict with the
    configuration of the url to contact the broker, overriding the trust
    bundle. Using forceSSLParameters overrides that behavior.
```

```
    [#1252] ci: fix tutorials flakiness
    
    Grafana is slower on github runners, and its password might not be
    prom-operator apparently. So fixing these two items to make the CI pass
    tests.
```

lavocatt commented on .github/workflows/ci.yml:
```diff
@@ -93,6 +93,7 @@ jobs:
         run: |
           helm repo add jetstack https://charts.jetstack.io --force-update
           helm upgrade -i -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true --create-namespace --wait
+          minikube kubectl -- wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=2m
```
"must show up" is showing up, but the chunk is still considered a success

brusdev commented on docs/tutorials/prometheus_locked_down.md:
```diff
@@ -1327,9 +1379,12 @@ Verify that the dashboard has been loaded correctly and has the expected configu
 echo ""
 
+# Get the Grafana admin password from the secret
+GRAFANA_PASSWORD=$(kubectl get secret -n locked-down-broker prometheus-grafana -o jsonpath="{.data.admin-password}" | base64 --decode)
```
The previous script already include a similar line to get GRAFANA_PASSWORD, can we export GRAFANA_PASSWORD to avoid a duplicate line?

lavocatt commented on docs/tutorials/prometheus_locked_down.md:
```diff
@@ -1126,8 +1139,44 @@ echo "  container_cpu_usage_seconds_total is queryable"
+  if [ $RETRY_COUNT -lt $MAX_RETRIES ]; then
+    sleep 5
+  fi
+done
```
I pushed a version with your suggestion, let's see if it pass :slightly_smiling_face:

lavocatt committed:
[#1252] ci: fix tutorials flakiness

Grafana is slower on github runners, and its password might not be
prom-operator apparently. So fixing these two items to make the CI pass
tests.

Fixes #1252:

lavocatt committed:
[#1264] make use of forceSSLParameters

On some environments JAVA env variables can conflict with the
configuration of the url to contact the broker, overriding the trust
bundle. Using forceSSLParameters overrides that behavior.
